### ISSUE TITLE
Is Facebook data scraper available

### ISSUE THREAD

**Issue by pj-mathematician** [neerajsinghsonu/Instagram-Data-Scraper#42]
Hello sir, I got your reference [here](https://stackoverflow.com/questions/19034754/facebook-api-search-for-hashtag?r=SearchResults) in the comments section, I am also in an urgent need of a similar application where we can find the number of posts with a particular hashtag on Facebook. Can you help me with it?
Also, sorry for writing to you here as I couldn't find any other way to reach you

**Comment by neerajsinghsonu** [comment]
Hi @pj-mathematician 

No worries :), let me share the my comments with you,

1. The Facebook Search API & Facebook Hashtag Search have been deprecated on April 30th, 2015.

2. From that day on, nobody can fetch the query and display public posts via hashtags from Facebook.

But, you can 

- Read your Facebook Pages & Page Albums to your wall and advise your users to post to your page’s timeline/brand page.

**TL;DR - it's not possible to fetch Facebook posts via a #hashtag any more. You can use Instagram, Twitter instead.**

Best of luck!
Neeraj Singh

**Comment by pj-mathematician** [comment]
Thanks alot for your reply, i think I have to shift to twitter and instagram only
